packages/rx_bloc_cli/mason_templates/bricks/feature_profile/__brick__/lib/l10n/network_lookup/profile_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nProfileLookup extends I18nFeatureProfileLookup {
           bundledI18n = I18nFeatureProfile(I18nFeatureProfileLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nFeatureProfile(I18nFeatureProfileLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong